### PR TITLE
CB-9330 Removes 'plugman publish' related functionality

### PR DESCRIPTION
**Pull request by vladimir-kotikov**
See [CB-9330](https://issues.apache.org/jira/browse/CB-9330) for more details.

### CODE REVIEW

File: cordova-lib/src/plugman/plugman.js [hunk 1]
```diff
@@ -169,35 +160,6 @@ plugman.commands =  {
             }
         });
     },
-
-    'publish'  : function(cli_opts) {
```
Comment (nikhilkh) [inline cordova-lib/src/plugman/plugman.js]: I wonder if these commands should still be exposed but with a meaningful console.error message on either having to use the older version of plugman < current or if we decide to make CPR read-only - stating that as such.

Comment (purplecabbage) [inline cordova-lib/src/plugman/plugman.js]: +1 to leaving the function stubs around for a bit and simply have them output that this functionality is not supported anymore.

**Comment by nikhilkh** [comment]
There is also an update to the docs that is needed.

**Comment by vladimir-kotikov** [comment]
@nikhilkh done. See https://github.com/apache/cordova-docs/pull/296

**Pull request closed without merging**